### ISSUE TITLE
Keyboard Shortcuts dialog: Generalizable solution when the dialog gets overloaded

### ISSUE THREAD

**Issue by terracoda** [phetsims/joist#960]
From https://github.com/phetsims/projectile-data-lab/issues/257#issuecomment-2032376142

While it has been the practice and is generally desirable to provide instructions for all non-Basic Actions, some sims present serious real estate challenges to achieve this goal.

Example sims with lots of information in their Keyboard Shortcuts dialogs:
- Ratio and Proportion (RaP)- Center and Variability (CaV)
- Project Data Lab (PDL)

For both these sims, there is a combobox interaction on each screen. We opted to only provide instructions for the combobox in Screen 1's dialog because there wasn't room in subsequent Screen dialogs.

This solution is not ideal and is not scalable when sims have even more controls and no more room in the dialog.

https://github.com/phetsims/projectile-data-lab/issues/257#issuecomment-2032376142 mentioned 2 potential general solutions to provide more room:
**Option 1: Implement Tabs**
- Put Basic Actions (or even General Actions) into a separate Tab in the dialog.
- Keep Sim Actions (the really important ones) on the first Tab (default Tab). This would make the KB Shortcuts dialog more like the Preferences Menu. This might be relatively straightforward since we already have a design pattern for an accessible Tab Menu in the Preferences Menu.

**Option 2: Central Resource for General/Basic Actions 
- The dialog would include a link to a general resource that describes General/Basic Actions and brief description of our general approach when there is not enough room to describe ever OR to keep the dialog from being overwhelming.
- The sim dialog, then would just contain Sim Actions (the really important ones) and sim-specific shortcuts.
- This approach would free up one column of space in the dialog.

This is a general design issue.

**Comment by terracoda** [comment]
Of course, a third solution is to try to keep the number of different types of controls on a sim screen to a minimum.

**Comment by jessegreenberg** [comment]
PhET does not have this, but another option could be to create a "scrollable" UI component for overlfowing content like this.

**Comment by kathy-phet** [comment]
Related to https://github.com/phetsims/scenery-phet/issues/854
